src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        isDone  = false;
+    }
+
+    public void doneIsFalse() {

```

Comment:
Consider renaming your setter methods. Perhaps you could change it to ```setDoneAsTrue()``` or something?